native_functions_test_metadata.yaml

```diff
@@ -0,0 +1,678 @@
+- func: __and__
```

Comment:
What's this?

Reply:
Not supposed to be part of PR. Removing.